src/app.js

```diff
@@ -1,21 +1,35 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  const normalizedUrl = new URL(req.url, `http://${req.headers.host}`);
+  const pathnameArray = normalizedUrl.pathname
+    .split('/')
+    .filter(path => path !== '');
+
+  if (pathnameArray[0] !== 'file') {
+    res.end('Correct request is: "/file/<path_to_file>"');
+
+    return;
+  }
+
+  const filename = pathnameArray.slice(1).join('/') || 'index.html';
+
+  fs.readFile(`./src/public/${filename}`, (err, data) => {
```

Comment:
You should verify if the requested file path contains '..' to ensure that the client cannot access files outside of the 'public' directory for security reasons.

Reply:
It already has implemented here:
<img width="437" alt="image" src="https://github.com/mate-academy/node_static-files-server/assets/25157075/63c7b651-2d6a-4464-9c5d-90d2a573339a">